hack/ovn-kind-cno.sh

```diff
@@ -0,0 +1,202 @@
+  sed -i '/host-run-netns/{n;s/mountPropagation.*/readOnly: true/}' bindata/network/ovn-kubernetes/ovnkube-node.yaml
+  if [ -z "$CNO_IMAGE" ]; then
+    echo "Error locating built CNO Image"
+    exit 1
+  fi
+  echo "Loading CNO image into KIND"
+  kind load docker-image $CNO_IMAGE --name ovn
+  popd
+fi
+
+if [ "$BUILD_OVN" = true ]; then
+  echo "Building OVN-K8S"
+  pushd $OVN_K8S_PATH
+  pushd go-controller
+  make clean
+  make
+  popd
+  pushd dist/images
+  sudo cp -f ../../go-controller/_output/go/bin/* .
+  cat << EOF | docker build -t origin-ovn-kubernetes:dev -f - .
+FROM quay.io/openshift/origin-ovn-kubernetes:4.3
+COPY ovnkube ovn-kube-util /usr/bin/
+COPY ovn-k8s-cni-overlay /usr/libexec/cni/ovn-k8s-cni-overlay
+COPY ovnkube.sh /root/
+COPY ovn-debug.sh /root/
+EOF
+  popd
+  echo "Loading OVN-K8S docker image into KIND"
+  kind load docker-image origin-ovn-kubernetes:dev --name ovn
+  popd
+fi
+
+NODES=$(docker ps | grep "kindest/node" | awk '{ print $1 }')
+for n in $NODES; do
+  echo "Modifying node $n"
+  echo "Copying kubeconfig"
+  docker cp ~/admin.conf $n:/etc/kubernetes/kubeconfig
+  docker exec $n chmod 777 /etc/kubernetes/kubeconfig
+  echo "Modifying os-release for Multus"
+  # required for Multus platform check
+  docker exec $n sed -i 's/ID=.*/ID=rhcos/' /etc/os-release
+done
+
+# Create Proxy resource
+kubectl create -f https://raw.githubusercontent.com/openshift/api/e7fa4b871a25985ef0cc36c2fbd9f2cb4445dc9c/config/v1/0000_03_config-operator_01_proxy.crd.yaml
+
+# Create Network resource
+kubectl create -f https://raw.githubusercontent.com/openshift/api/e7fa4b871a25985ef0cc36c2fbd9f2cb4445dc9c/config/v1/0000_10_config-operator_01_network.crd.yaml
+
+# Create cluster operator
+kubectl create -f https://raw.githubusercontent.com/openshift/machine-api-operator/050a65a2bdabcc2c2f17036de967c6bcee6d6a48/config/0000_00_cluster-version-operator_01_clusteroperator.crd.yaml
+
+if [ "$BUILD_OVN" = true ] || [ "$BUILD_CNO" = true ]; then
+  pushd $CNO_TEMPLATES
+  DEPLOYMENT_TEMPLATE=$(ls 0000*deployment*)
+  if [ -z "$DEPLOYMENT_TEMPLATE" ]; then
+    echo "error locating deployment template in $CNO_TEMPLATES"
+    exit 1
+  fi
+  cp $DEPLOYMENT_TEMPLATE deployment.yaml.bk
+  if [ "$BUILD_OVN" = true ]; then
+    sed -i 's/".*origin-ovn-kubernetes:.*/"origin-ovn-kubernetes:dev"/' $DEPLOYMENT_TEMPLATE
+  fi
+  if [ "$BUILD_CNO" = true ]; then
+    sed -i "s#quay.io/openshift/origin-cluster-network-operator:.*#$CNO_IMAGE#" $DEPLOYMENT_TEMPLATE
+  fi
+fi
+
+echo "Creating CNO operator"
+for f in $(ls $CNO_TEMPLATES| grep 0000| grep -v credentials); do
+  kubectl create -f ${CNO_TEMPLATES}/$f
+done
+
+if [ "$BUILD_OVN" = true ] || [ "$BUILD_CNO" = true ]; then
+  mv deployment.yaml.bk $DEPLOYMENT_TEMPLATE
+  popd
+fi
+
+count=1
+until kubectl get pod -n openshift-network-operator -o jsonpath='{.items[0].metadata.name}' --field-selector status.phase=Running &> /dev/null ;do
```

Comment:
reachability check fails here without a rule in iptables to allow the pod to reach out to the api server

Reply:
thanks I will add a rule to allow traffic in the Docker table.